### PR TITLE
Sanitize user input in workflows

### PR DESCRIPTION
**Pull request by rth**
Adds very light input sanitizing of user submissions to make aware that poking around the RAMP server is not recommended.

For sensitive discussions please use email.

The actual long term solution is https://github.com/paris-saclay-cds/ramp-board/pull/374 but it's still useful to detect attempts and make users aware I think. Docker is not completely fullproof either https://resources.whitesourcesoftware.com/blog-whitesource/top-5-docker-vulnerabilities

### CODE REVIEW

File: rampwf/utils/importing.py [hunk 1]
```diff
@@ -3,9 +3,10 @@
 
 """
 import importlib
+from .sanitize import _sanitize_input
 
 
-def import_module_from_source(source, name):
+def import_module_from_source(source, name, sanitize=False):
```
Comment (rth) [inline rampwf/utils/importing.py]: Can't make `True` the default, as it's also used in various notebooks to load `problem.py`

**Comment by rth** [comment]
Great. If there is no objection can someone merge then?

File: rampwf/utils/importing.py [hunk 2]
```diff
@@ -14,12 +15,18 @@ def import_module_from_source(source, name):
         Path to the Python source file which will be loaded as a module.
     name : str
         Name to give to the module once loaded.
+    sanitize: bool, default=False
+        check for blacklisted key words in code
```
Comment (glemaitre) [inline rampwf/utils/importing.py]:
```suggestion
        Check for blacklisted key words in code.
```

**Comment by glemaitre** [comment]
Actually there is not what's new in ramp-workflow

**Comment by glemaitre** [comment]
Thanks @rth

**Pull request merged**